microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/ls/api/MicroProfileJavaCodeActionProvider.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.

```

Comment:
Do we need the 2020 header and "SPDX-License-Identifier: EPL-2.0" in this file?

Reply:
fixed